### PR TITLE
Add dedicated download button for Apple Silicon

### PR DESCRIPTION
**Pull request by leo**
Follow-up to https://github.com/vercel/hyper-site/pull/253.

Inspired by Google Chrome:

CleanShot 2021-09-14 at 17 04 03@2x

### CODE REVIEW

**Comment by vercel-team** [comment]
#### 📝 Changed routes:


| [**/**](https://hyper-site-iuroipzp7.vercel.sh/) |
|:-:|
|<a href="https://hyper-site-iuroipzp7.vercel.sh/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/RPYI28oBSEMOhkwhf5AP3hqH0H_RdVnTNBelsrE2tflR4Auup0pVttGje11X0dbpSJfwJKKSdsSAy1FRkOt2Lw.png">(view full size)</a> |







Commit 7b44df242c7e330c8684e34ee1b0dbd681f2194b (https://hyper-site-iuroipzp7.vercel.sh).

File: components/download-button/index.js [hunk 1]
```diff
@@ -7,15 +7,26 @@ const DownloadButton = () => {
...
-        <strong>Download Hyper</strong>
-      </a>
+      <div className={styles.doubleWrap}>
+        <a
+          className={styles.root}
+          href="https://releases.hyper.is/download/mac"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          <Apple size={16} />
```
Comment (leerob) [inline components/download-button/index.js:17]: This is wrong I think.

**Review by leerob — COMMENTED** [review]
Should we switch to npm fully here and remove Yarn?

**Comment by leo** [comment]
> Should we switch to npm fully here and remove Yarn?

Adjusted the readme – that was the only remaining piece.

**Pull request merged**